File: src/main/java/busanVibe/busan/domain/user/dto/UserResponseDTO.java
package busanVibe.busan.domain.user.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;

public class UserResponseDTO {

  @Getter
  @AllArgsConstructor
  public static class loginDto {
    private Long id;
    private TokenResponseDTO tokenResponseDTO;
    private String email;
    private boolean isNewUser;
  }

}

File: src/main/kotlin/busanVibe/busan/domain/user/data/dto/UserResponseDTO.kt
package busanVibe.busan.domain.user.data.dto

import com.fasterxml.jackson.databind.PropertyNamingStrategies
import com.fasterxml.jackson.databind.annotation.JsonNaming

class UserResponseDTO {

    @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
    data class MyPageDto(
        val nickname: String,
        val email: String,
        val userImageUrl: String? = null
    )

}